problems/concat/solution.js

```diff
@@ -4,5 +4,5 @@
   
   process.stdin.pipe(concat(function (src) {
       var s = src.toString().split('').reverse().join('');
-      console.log(s);
+      process.stdout.write(s);
```

Comment:
the difference between `process.stdout.write`and `console.log` is that console.log adds a line break after so you want to add one here as well